pkgs/top-level/all-packages.nix

```diff
@@ -38848,7 +38848,7 @@ with pkgs;
 
   spacenav-cube-example = callPackage ../applications/misc/spacenav-cube-example { };
 
-  spnavcfg = callPackage ../applications/misc/spnavcfg { };
+  spnavcfg = libsForQt5.callPackage ../applications/misc/spnavcfg { };
```

Comment:
As I commented on the `spnavcfg` package, I'd much rather that the package contain the details about which version of qt is needed than `all-packages.nix`.